src/plone/restapi/serializer/dxcontent.py

```diff
@@ -39,6 +39,14 @@
     WorkingCopyInfo = None
 
 
+def get_allow_discussion_value(context, request, result):
+    # This test is to handle the plone.app.discussion not being installed situation
+    if "allow_discussion" in result:
+        result["allow_discussion"] = getMultiAdapter(
+            (context, request), name="conversation_view"
+        ).enabled()
+
```

Comment:
I'd recommend returning False even when plone.app.discussion isn't installed, so that this isn't a backwards incompatible change.

```suggestion
    else:
        return False

```

Reply:
@Akshat2Jain @davisagli In this case, the `allow_discussion` key would have to be created:

```suggestion
    else:
        result["allow_discussion"] = False
```

Reply:
@Akshat2Jain see my comment above. It should fix the [failing test](https://github.com/plone/plone.restapi/actions/runs/6899768072/job/18771880100?pr=1674#step:10:228).